### DISCUSSION TITLE
Why I got blank page after building

### DISCUSSION THREAD

**Discussion by venusbai** [kingyue737/vitify-admin#26] [Q&A]
I can run the site in dev mode, but after building it, the preview site only shows a blank (black) page. I also tried to deploy the contents in the dist folder to a web server, but still got a blank page. I wonder what steps I missed. Thanks for any response.

**Comment by kingyue737** [comment]
Strange, you don't need any further steps. Can you open the [demo](https://vitify-admin.netlify.app/) site? Netlify shows that it is built successfully and not blank.
![image](https://github.com/kingyue737/vitify-admin/assets/40021217/0579d721-7d07-4c45-900c-fba6c2409516)

Anything printed in your browser console?
